R/nneo_wrangle.R

```diff
@@ -0,0 +1,108 @@
+#############################################################################################
+#' @title Download and wrangle NEON's sensor-based data products
+
+#' @author Josh Roberti \email{jaroberti87@gmail.com}\cr
```

Comment:
@sckott.  Interesting,  I was always under the impression it was a single '@'.  I notice in nneo_data and nneo_sites, for example, that a single '@' is used.   Is the double '@' only for certain roxygen tags, e.g., title and author?